src/test/java/org/jabref/logic/importer/fetcher/IacrEprintFetcherTest.java

```diff
@@ -0,0 +1,119 @@
+package org.jabref.logic.importer.fetcher;
+
+import java.util.Optional;
+
+import org.jabref.logic.importer.FetcherException;
+import org.jabref.logic.importer.ImportFormatPreferences;
+import org.jabref.model.entry.BibEntry;
+import org.jabref.model.entry.BiblatexEntryTypes;
+import org.jabref.model.entry.FieldName;
+import org.jabref.testutils.category.FetcherTest;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.mockito.Answers;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.Mockito.mock;
+
+@FetcherTest
+public class IacrEprintFetcherTest {
+
+    private static IacrEprintFetcher fetcher;
```

Comment:
Please not static and use `BeforeEach` instead `BeforeAll` (better initialize the fetcher and entries fresh for each test)

Reply:
Thanks, it's my first time using JUnit 5 and I just looked at some examples online which used `BeforeAll`...
Should have been suspicious when adding the `static`s...